packages/nodes-base/nodes/Wise/GenericFunctions.ts

```diff
@@ -65,6 +70,95 @@ export async function wiseApiRequest(
 	}
 }
 
+/**
+ * Make an authenticated SCA API request to Wise.
+ */
+ export async function wiseSCAApiRequest(
```

Comment:
> This probably doesn't strictly require a new `wiseSCAApiRequest` function, we can probably just check for SCA failure regardless of the request and cut down on duplicated code.

Agree

Reply:
Merged the functions. As far as I can tell the request helper just returns `response.body`, so using the `httpRequest` helper and returning `response.body` is equivalent.